src/maintainers/CommunityContributionMaintainer.ts

```diff
@@ -30,6 +29,25 @@ export default class CommunityContributionMaintainer extends BaseMaintainer {
     try {
       const connection = await this.db.connect();
 
+      // check if local executable file is git
```

Comment:
Why was this code added here? Just to check the manifest?

Reply:
earlier this code snipped used to live between update job and submit job sections but since we are extracting Connector out of manifest it had to move here to update the connector state before it is used.